main/res/drawable/ic_menu_sync_enabled.xml

```diff
@@ -0,0 +1,11 @@
+<!-- taken from sync-disabled / material.io -->
```

Comment:
Thanks for your contribution!

"sync-disabled" doesn't sound right for the used icon. 

You can use the official material icon from here:
https://fonts.google.com/icons?selected=Material%20Icons%3Async%3A